yoshi/yoshi.yaml

```diff
@@ -0,0 +1,581 @@
+apiVersion: v1
```

Comment:
This seems like a generated file, right? If so, it should not be checked in

Reply:
I will move it to luigi/samples folder where I keep other examples and manifests